api/src/main/java/uk/gov/dhsc/htbhf/hmrc/service/EligibilityService.java

```diff
@@ -46,8 +36,8 @@ public EligibilityResponse checkEligibility(HMRCEligibilityRequest eligibilityRe
             return getEligibilityResponse(eligibilityRequest, household.get());
         }
 
-        log.debug("No match found in db - calling HMRC to check eligibility");
-        return getResponseFromHMRC(eligibilityRequest);
+        log.debug("No match found in db - returning NO_MATCH");
```

Comment:
I think this should be at a higher level now because during the beta this means that someone has applied who isn't in our database and we probably want to keep track of these, so I'd say at least INFO level.